pages/index.tsx

```diff
@@ -16,6 +17,37 @@ export default function Home() {
           <a>shipping may be delayed</a>
         </Link>
       </div>
+
+      <div className="home-page__info gel-layout">
+        <div className="home-page__info__section gel-layout__item gel-1/3">
+          <img className="home-page__info__image" src="/svgs/sparkles.svg" alt="" />
+          <p className="home-page__info__text">Unique card designs that you won't find anywhere else</p>
+        </div>
+        <div className="home-page__info__section gel-layout__item gel-1/3">
+          <img className="home-page__info__image" src="/svgs/email-dark.svg" alt="" />
+          <p className="home-page__info__text">Direct to recipient shipping available to any UK address</p>
+        </div>
+        <div className="home-page__info__section gel-layout__item gel-1/3">
+          <img className="home-page__info__image" src="/svgs/sustainability.svg" alt=""/>
+          <p className="home-page__info__text">Committed to high quality and sustainable products</p>
+        </div>
+      </div>
+
+      <div className="home-page__cta gel-layout">
+        <a className="home-page__cta__wrapper gel-layout__item gel-1/2@m gel-1/3@l" href="/cards">
+          <img className="home-page__cta__image" src="/featured/all-cards.jpg" />
```

Comment:
Want some `alt` on this and `aria-label` on the links?

Reply:
Have added blank alt text.